virtualbox/create-vm.sh

```diff
@@ -75,3 +77,9 @@ echo "######### start vm and configure SSH Port forward"
 vboxmanage startvm ${VIRTUAL_BOX_NAME} --type headless
 vboxmanage controlvm ${VIRTUAL_BOX_NAME} natpf2 ssh,tcp,127.0.0.1,5222,,22
 
+if [ -x "$(command -v ansible-playbook)" ]; then
```

Comment:
Have you documented how to create vm and generate inventory ?

Reply:
There is really nothing more for me to document I think, you have already documented how to create a VM with the two scripts.

I did however add a note in `ansible/README-oc.md` that generating the inventory is no longer necessary if the target VM was created with those scripts. See https://github.com/snowdrop/openshift-infra/blob/884dc286670e6467e34e40705f4d19b417079973/ansible/README-oc.md

Reply:
What is the purpose of the command `$(command -v ansible-playbook)` ?

Reply:
It tests whether `ansible-playbook` is a valid "executable". 
It's basically a safeguard to allow to script to work even when ansible is not installed

Reply:
> There is really nothing more for me to document I think

This part of the doc should be then updated to mention that - https://goo.gl/xMhRYW

Reply:
That's a good idea, I'll do that

Reply:
Doc updated